src/components/SingleArticle.jsx

```diff
@@ -39,6 +42,22 @@ export const SingleArticle = () => {
                     )
                 })}
             </ul>
+
+            <ul id="commentsUnorderedList">
```

Comment:
There'd be a lot of value in breaking comments out into it's own component - single article is getting a bit busy, and they're separate concerns, really.